libs/api/domains/watson-assistant-chat/src/lib/dto/watsonAssistantChatIdentityToken.input.ts

```diff
@@ -0,0 +1,13 @@
+import { Field, InputType } from '@nestjs/graphql'
+
+@InputType()
+export class WatsonAssistantChatIdentityTokenInput {
```

Comment:
FYI you can also create the class and the file with shorter names and use the GQL decorator to control the schema name like:
```
@InputType('WatsonAssistantChatIdentityTokenInput')
export class IdentityTokenInput {
```

This is possible for the `ObjectType`, `InputType`, `Mutation`.  Just up to you what you prefer.